### PR TITLE
dbimport: add debug option, will sort files by modification date

### PR DESCRIPTION
**Pull request by Athanasiamo**

### CODE REVIEW

**Review by f-krull — APPROVED** [review]
nice and short :+1:

File: dbimport/main.R [hunk 1]
```diff
@@ -55,6 +55,8 @@ fail_if(length(core_files) > 0,
 # import non-core
 ncore_dir <- file.path(getOption("noas")$TABDIR, "non_core")
 table_ids <- list.dirs(ncore_dir, recursive = FALSE, full.names = FALSE)
+if(getOption("noas")$IMPORT_DEBUG == 1)
```
Comment (f-krull) [inline dbimport/main.R:58]: maybe `$IMPORT_DEBUG != "0"` is safer?

**Review by f-krull — APPROVED** [review]

File: dbimport/main.R (hunk 1 above)
Comment (f-krull) [reply dbimport/main.R:58]: It could be that all configs are strings by default

Comment (Athanasiamo) [reply dbimport/main.R:58]: isnt it safer to have a very explicit debugging flag, and everything else is interpreted as business as usual?

Comment (f-krull) [reply dbimport/main.R:58]: yes, I think thats fine too. I was doing two things at the same time. My main point was the quotes

Comment (f-krull) [reply dbimport/main.R:58]: `$IMPORT_DEBUG == 1` => `$IMPORT_DEBUG == "1"`

Comment (Athanasiamo) [reply dbimport/main.R:58]: oh yeah. quotes sounds good

**Review by f-krull — APPROVED** [review]

**Pull request merged**